asynctnt/const.py

```diff
@@ -0,0 +1,3 @@
+class Transport():
```

Comment:
As I described earlier - maybe it'll be better to name it ServerTransport or maybe something else to avoid confusion with asyncio.Transport. And also it is better to make it an `Enum` class.

Reply:
On `tarantool` side, server is set with `listen = { uri = 'localhost:3301', params = { transport = 'ssl' }` or `listen = 'localhost:3301?transport=ssl', client is set with `net_box.connect({ uri = 'localhost:3301', params = {transport = 'ssl'}})` or  `net_box.connect({ uri = 'localhost:3301?transport=ssl'})`. Synchronous Python connector uses `transport` connection argument to set up SSL. I think it would be the best to call this parameter with plain `transport` at least on the outside. As far as I understand, `asyncio.Transport` is build and used internally -- user have no way or reason to pass or use it from the outside.

Reply:
https://www.tarantool.io/en/enterprise_doc/security/#configuration

Reply:
ok, let's leave a `Transport` name but please consider making it an Enum

Reply:
Used enum